src/NuGet.Core/NuGet.Protocol/PublicAPI/net472/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,6 @@
+NuGet.Protocol.Core.Types.PackageSearchMetadataBuilder.ClonedPackageSearchMetadata.PackagePath.get -> string
+NuGet.Protocol.Core.Types.PackageSearchMetadataBuilder.ClonedPackageSearchMetadata.PackagePath.set -> void
```

Comment:
I've mentioned this to you before, but I really don't like `PakageSearchMetadataBuilder` and `ClonedPackageSearchMetadata`. Technically it's out of scope for your PR, but I'm obsessed with trying to get rid of this from our public API.

So, I looked at references to `PackageSearchMetadataBuilder`, and see there's only 2 places it's used outside of tests. PM UI and CommandLine.XPlat. CommandLine.XPlat's usage is just to create a `IPackageSearchMetadata` that only has a package id and version, nothing else. It should be easily replaceable with any other implementation of `IPackageSearchMetadata`.

Therefore, `PackageSearchMetadata` and `ClonedPackageSearchMetadata` is really just PM UI's abstraction (used particularly for the multi feed aggregator), so I strongly feel this code should be "moved" to one of the `src\NuGet.Clients` VS projects. Any way I can convince you to do that and mark NuGet.Protocol's `PackageSearchMetadataBuilder` as obsolete?

Reply:
Sorry, time is short right now, and we'll be losing some of our resources, so I'd like to keep to the minimum work.